main.tf

```diff
@@ -215,4 +263,17 @@ module "graphdb" {
   asg_enable_instance_refresh               = var.asg_enable_instance_refresh
   asg_instance_refresh_checkpoint_delay     = var.asg_instance_refresh_checkpoint_delay
   graphdb_enable_userdata_scripts_on_reboot = var.graphdb_enable_userdata_scripts_on_reboot
+
+  # Parameter store encryption
+
+  enable_graphdb_parameter_store_kms_key              = var.create_graphdb_ebs_kms_key
```

Comment:
This should be named `create_graphdb_parameter_store_kms_key` so it's consistent with the rest.